internal/charm/styles.go

```diff
@@ -1,4 +1,4 @@
-package styles
+package charm
```

Comment:
I feel like in this renaming the rest of the code that uses the styles doesn't read as nicely. `styles.Focused` for example makes immediate sense, whereas `charm.Focused` is confusing. Can we just move the styles specifically under their own subpackage?

Reply:
I originally had styles under charm but didn't like the sub package nesting. I wanted to try to isolate as many imports of charm as possible into the charm wrapper.

Thoughts on charm/styles vs styles?

Reply:
No strong preference, happy with either